cmd/minishift/cmd/addon/addon_apply.go

```diff
@@ -75,15 +77,20 @@ func runApplyAddon(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting IP: %s", err.Error()))
 	}
 
-	ocPath := minishiftConfig.InstanceConfig.OcPath
 	routingSuffix := viper.GetString(routingSuffix)
-	kubeConfigPath := constants.KubeConfigPath
 	sshCommander := provision.GenericSSHCommander{Driver: host.Driver}
+	ocRunner, err := oc.NewOcRunner(minishiftConfig.InstanceConfig.OcPath, constants.KubeConfigPath)
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error applying addon: %s", err.Error()))
+	}
 
 	for i := range args {
 		addonName := args[i]
 		addon := addOnManager.Get(addonName)
-		addonContext := GetExecutionContext(ip, routingSuffix, ocPath, kubeConfigPath, sshCommander)
+		addonContext, err := clusterup.GetExecutionContext(ip, routingSuffix, ocRunner, sshCommander)
+		if err != nil {
+			atexit.ExitWithMessage(1, fmt.Sprint("Error executing addon commands: ", err))
```

Comment:
@LalatenduMohanty you can actually write `"Hello", "World", "!"`
https://play.golang.org/p/3Mqog0waoH